lib/utils/string_extension.dart

```diff
@@ -306,4 +306,8 @@ extension StringCasingExtension on String {
     }
     return this;
   }
+  
+  String getFileNameFromPath() {
```

Comment:
please write unittest for that
it also crash if split result empty

Reply:
sr i don't use this function, so remove it :)